packages/xml-views-validation/src/utils/custom-class.ts

```diff
@@ -0,0 +1,36 @@
+import { SVG_NS, TEMPLATING_NS, XHTML_NS } from "./special-namespaces";
+import { XMLElement } from "@xml-tools/ast";
+import { resolveXMLNS } from "@ui5-language-assistant/logic-utils";
+
+// Heuristic to limit false positives by only checking tags starting with upper
+// case names, This would **mostly** limit the checks for things that can actually be
+// UI5 Elements / Controls.
+export function isCustomClass(xmlElement: XMLElement): boolean {
+  // This will never happen - it was checked before calling this function
```

Comment:
> // This will never happen - it was checked before calling this function

We now use this function in multiple places, its a mini API.
I don't think we can still make this assumption about the null value.
But perhaps we can model it in the signature instead.

```typescript
export function isCustomClass(xmlElement: XMLElement & { name: string}): boolean 
```